### ISSUE TITLE
Gutter icon in ml file pointing to mli file disappears when it shouldn't

### ISSUE THREAD

**Issue by jfehrle** [giraud/reasonml-idea-plugin#433]
plugin version: 0.117.1-2023.1

### Description

Adding a space after the `=` makes the gutter icon disappear.  To get it to reappear, I need to reopen the editor panel for that file OR switch to another window and back.  This may be an IDEA bug rather than a ReasonML bug.

I'm using Windows 10 FWIW.

![image](https://github.com/giraud/reasonml-idea-plugin/assets/1253341/0cfea103-1b87-4abd-adf0-60655e2dde99)

![image](https://github.com/giraud/reasonml-idea-plugin/assets/1253341/57b81fb0-9d1d-4b30-9302-8cab738ca0a4)

```
let diff_pp_combined x =
  let open Pp in
  x

val diff_pp_combined : int -> int
```

**Comment by giraud** [comment]
The `let open` has a side effect when collecting markers.

**Comment by giraud** [comment]
fixed in 0.118

**Comment by jfehrle** [comment]
Looks good, thanks!
